src/c++/library/grpc_client.cc

```diff
@@ -1347,6 +1369,8 @@ InferenceServerGrpcClient::PreRunProcessing(
   infer_request_.set_id(options.request_id_);
 
   infer_request_.mutable_parameters()->clear();
+  (*infer_request_.mutable_parameters())["triton_enable_empty_final_response"]
```

Comment:
This is in the grpc client. Wouldn't this change affect everyone and not just PA?

Reply:
Good point. It's now passed in via InferOptions from PA, and default is `false` 👍